client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -108,7 +133,14 @@ const ProductLightbox: React.FC< Props > = ( {
 		} ) );
 	}, [ product.productSlug ] );
 
-	const shouldShowOptions = variantOptions.length > 1;
+	const isSocialProduct = [
+		PRODUCT_JETPACK_SOCIAL_ADVANCED,
+		PRODUCT_JETPACK_SOCIAL_BASIC,
+	].includes( product.productSlug );
+
+	const shouldShowOptions = isSocialProduct
+		? variantOptions.length > 1 && config.isEnabled( 'jetpack-social/advanced-plan' )
+		: variantOptions.length > 1;
```

Comment:
I'm guessing this is temporary because the Advanced plan isn't available yet, right? If so, maybe we can add a comment here and add an Asana task to remove it once the plan is launched.